Clicker/Views/PollBuilder/AskQuestionCell.swift

```diff
@@ -12,55 +12,84 @@ import UIKit
 protocol AskQuestionCellDelegate: class {
     func askQuestionCellDidUpdateEditable(_ editable: Bool)
     func askQuestionCellDidUpdateText(_ text: String?)
+    func askQuestionCellDidUpdateHeight(_ height: CGFloat)
 }
 
 class AskQuestionCell: UICollectionViewCell {
 
     // MARK: - View vars
-    var textField: UITextField!
+    var questionTextView: UITextView!
+    var charCountLabel: UILabel!
 
     // MARK: - Data vars
     weak var delegate: AskQuestionCellDelegate?
-
+    
     // MARK: - Constants
     let editButtonImageName = "dots"
     let moreButtonWidth: CGFloat = 25
+    let questionLabelPlaceholder = "Ask a question..."
     let questionLabelWidthScaleFactor: CGFloat = 0.75
+    let charCountLabelPadding: CGFloat = 4
     let untitledPollString = "Untitled Poll"
 
     override init(frame: CGRect) {
         super.init(frame: frame)
 
         contentView.backgroundColor = .clickerWhite
-
-        textField = UITextField()
-        textField.attributedPlaceholder = NSAttributedString(string: "Ask a question...", attributes: [NSAttributedStringKey.foregroundColor: UIColor.clickerGrey2, NSAttributedStringKey.font: UIFont._18RegularFont])
-        textField.font = ._18RegularFont
-        textField.returnKeyType = .done
-        textField.delegate = self
-        textField.addTarget(self, action: #selector(updateEditable), for: .allEditingEvents)
-        textField.addTarget(self, action: #selector(textDidChange), for: .editingChanged)
-        contentView.addSubview(textField)
-
-        textField.snp.makeConstraints { make in
-            make.edges.equalToSuperview()
-        }
+        
+        questionTextView = UITextView()
+        questionTextView.delegate = self
+        questionTextView.font = ._18RegularFont
+        questionTextView.returnKeyType = .done
+        questionTextView.sizeToFit()
```

Comment:
is this line necessary if you're setting up constraints later?

Reply:
Removed the sizeToFit()